jms/src/main/java/com/sun/ts/tests/jms/ee20/cditests/ejbweb/MyManagedBean.java

```diff
@@ -38,149 +39,147 @@
 
 @ManagedBean(value = "mybean")
```

Comment:
Not related to this refactor change ; Gentle reminder when testing this tests in EE11 `@jakarta.annotation.ManagedBean` will need to be removed (https://github.com/jakartaee/platform-tck/issues/1184). IIUC `@jakarta.inject.Named` can be used instead.